configs-dev.json

```diff
@@ -0,0 +1,80 @@
+{
+  "total": 18,
+  "limit": 18,
+  "offset": 0,
+  "data": [
+    {
+      "key": "commerce.headers.cs.Magento-Customer-Group",
+      "value": "b6589fc6ab0dc82cf12099d1c2d40ab994e8410c"
+    },
+    {
+      "key": "commerce.headers.cs.Magento-Environment-Id",
+      "value": ""
+    },
+    {
+      "key": "commerce.headers.cs.Magento-Store-Code",
+      "value": "main_website_store"
+    },
+    {
+      "key": "commerce.headers.cs.Magento-Store-View-Code",
+      "value": "default"
+    },
+    {
+      "key": "commerce.headers.cs.Magento-Website-Code",
+      "value": "base"
+    },
+    {
+      "key": "commerce.headers.cs.x-api-key",
+      "value": ""
+    },
+    {
+      "key": "commerce-base-currency-code",
+      "value": "USD"
+    },
+    {
+      "key": "commerce-core-endpoint",
+      "value": "https://edge-graph.adobe.io/api/1ed11f0b-4dc6-4029-8e4d-631fafb4dec4/graphql"
```

Comment:
This file is just a way to force the override of adobe-demo-store site to use this commerce endpoint. In partner day, each person will have their own site + config, pointing to their own mesh already.